charts/scalardb-cluster-monitoring/templates/scalardb-cluster-monitoring-stack/configmap.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: scalardb-cluster-grafana-dashboard
+  namespace: {{ .Release.Namespace }}
+  labels:
+    grafana_dashboard: "1"
+    {{- include "scalardb-cluster-monitoring.labels" . | nindent 4 }}
+data:
+{{- range $path, $_ :=  .Files.Glob  "files/grafana/scalardb_cluster_*.json" }}
+  {{ $path | base }}: |-
+{{ $.Files.Get $path | indent 4 }}
+{{- end }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: scalar-envoy-grafana-dashboard
+  namespace: {{ .Release.Namespace }}
+  labels:
+    grafana_dashboard: "1"
+    {{- include "scalardb-cluster-monitoring.labels" . | nindent 4 }}
```

Comment:
FYI: If you want to include the dashboard in a specific folder, you can use the `sidecar.dashboards.folderAnnotation` setting in the Grafana chart.

- https://github.com/grafana/helm-charts/blob/main/charts/grafana/README.md

Reply:
Oh, I see. I didn't know that. I will consider using it in the future if necessary. Thank you!